src/App.js

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, {useState} from 'react';
+import {Routes, Route} from 'react-router-dom';
+
 import './App.css';
 
-function App() {
-  return (
-      <>
+import Home from './pages/home/Home';
+import Subreddit from './pages/subreddit/Subreddit'
+import Header from './components/header/Header';
+import Footer from './components/footer/Footer';
 
-      </>
-  );
+
+function App() {
+    const [subreddit, setSubreddit] = useState();
+    return (
+        <>
+            <header className="outer-container">
+                <Header subredditState={subreddit}/>
+            </header>
```

Comment:
Het is goed dat je de header hier hebt geplaatst. Je wilt zo min mogelijk code dupliceren. Je zou bijna kunnen zeggen dat wanneer iets óver meerdere componenten heen staat (niet letterlijk), het eigenlijk "gelift" moet worden naar "boven". Deze plek is hogerop in de DOM tree en staat dus letterlijk en figuurlijk bóven die componenten. 
Dus het staat hier op de goeie plek! 

MAND! 🤣